pkg/scheduler/framework/plugins/tainttoleration/plugin.go

```diff
@@ -0,0 +1,64 @@
+package tainttoleration
+
+import "go.goms.io/fleet/pkg/scheduler/framework"
+
+// Plugin is the scheduler plugin that checks to see if taints on the MemberCluster
+// can be tolerated by tolerations on ClusterResourcePlacement
+type Plugin struct {
+	// The name of the plugin.
+	name string
+
+	// The framework handle.
+	handle framework.Handle
+}
+
+var (
+	// Verify that Plugin can connect to relevant extension points at compile time.
+	//
+	// This plugin leverages the following the extension points:
+	// * Filter
+	//
+	// Note that successful connection to any of the extension points implies that the
+	// plugin already implements the Plugin interface.
+	_ framework.FilterPlugin = &Plugin{}
+)
+
+type taintTolerationPluginOptions struct {
+	// The name of the plugin.
+	name string
+}
+
+type Option func(*taintTolerationPluginOptions)
+
+var defaultPluginOptions = taintTolerationPluginOptions{
+	name: "TaintToleration",
+}
+
+// WithName sets the name of the plugin.
+func WithName(name string) Option {
+	return func(o *taintTolerationPluginOptions) {
+		o.name = name
+	}
+}
+
+// New returns a new Plugin.
+func New(opts ...Option) Plugin {
```

Comment:
you will add this plugin in your next PR?

Reply:
Yes